### PR TITLE
Add GTK icon

### PR DESCRIPTION
**Pull request by zrfisaac**
gtk

**Issue:** closes #

**Similarweb rank:** [258,780](https://www.gtk.org/)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
GTK (formerly GIMP ToolKit then GTK+)[[4]](https://en.wikipedia.org/wiki/GTK#cite_note-:0-4) is a [free and open-source](https://en.wikipedia.org/wiki/Free_and_open-source) [cross-platform](https://en.wikipedia.org/wiki/Cross-platform) [widget toolkit](https://en.wikipedia.org/wiki/Widget_toolkit) for creating [graphical user interfaces](https://en.wikipedia.org/wiki/Graphical_user_interface) (GUIs).[[5]](https://en.wikipedia.org/wiki/GTK#cite_note-5) It is licensed under the terms of the [GNU Lesser General Public License](https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License), allowing both [free](https://en.wikipedia.org/wiki/Free_software) and [proprietary software](https://en.wikipedia.org/wiki/Proprietary_software) to use it. It is one of the most popular toolkits for the [Wayland](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)) and [X11](https://en.wikipedia.org/wiki/X_Window_System_core_protocol) [windowing systems](https://en.wikipedia.org/wiki/Windowing_system).[[6]](https://en.wikipedia.org/wiki/GTK#cite_note-6)

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4331,6 +4331,11 @@
             "source": "https://github.com/gruntjs/gruntjs.com/blob/70f43898d9ce8e6cc862ad72bf8a7aee5ca199a9/src/media/grunt-logo-no-wordmark.svg",
             "guidelines": "https://github.com/gruntjs/grunt-docs/blob/main/Grunt-Brand-Guide.md"
         },
+        {
+            "title": "GTK",
+            "hex": "7FE719",
+            "source": "https://upload.wikimedia.org/wikipedia/commons/7/71/GTK_logo.svg"
```
Comment (sisp) [inline _data/simple-icons.json:4337]: Please use the Wikipedia link to the logo file's _page_ (see [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines)) and add the link to the brand guidelines:

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:GTK_logo.svg",
            "guidelines": "https://foundation.gnome.org/logo-and-trademarks/"
```

File: icons/gtk.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GTK</title><path d="M13.695 0v8.065l8.93-2.748zm-.325.09L1.415 3.372 9.423 9.38l3.946-1.215zM.76 4.112l.494 12.366 7.502-2.22v-4.148zm22.482 1.933-9.546 2.938v3.945l8.52 5.938zM13.37 9.084l-3.475 1.07v3.768l3.475-1.03zm.134 4.108-3.61 1.068V24l11.965-4.984zm-4.748 1.405-7.488 2.217 7.488 6.99z"/></svg>
```
Comment (sisp) [inline icons/gtk.svg:1]: When I derive the icon from the Wikipedia logo, I see a notable difference between yours (blue) and mine (red):

gtk-comparison

I converted the strokes to paths, disabled the strokes on the original shapes, and subtracted the stroke paths from the original shapes. Then, I created a union of the paths, resized them, and centered the path.

Could you please double-check?

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GTK</title><path d="M9.01 23.773V14.45l-7.584 2.245Zm0-13.87L.91 3.828l.502 12.526 7.597-2.249ZM9.57 24l12.353-5.146-8.285-5.775-4.068 1.204ZM23.09 5.815l-9.257 2.849v4.148l8.237 5.741ZM9.57 9.975v3.964l3.932-1.164v-4.01Zm-.228-.52 4.16-1.28V0L1.231 3.37ZM22.715 5.34 13.833.052v8.021Z"/></svg>
```

**Review by sisp — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (zrfisaac) [reply _data/simple-icons.json:4337]: Thanks, to me looks good! 🙌

Comment (sisp) [reply _data/simple-icons.json:4337]: This suggested change is still pending. If you agree, could you please add a commit?

**Comment by zrfisaac** [comment]
Sorry, I commited now! 😅

**Review by sisp — APPROVED** [review]
LGTM now! :tada: Thanks for making the requested changes.

This PR can be merged IMO.

/cc @sachinraja

**Comment by sachinraja** [comment]
Just letting you know @zrfisaac so you're not wasting time, we don't need a description of what the tool/brand does. We just need to know details related to assessing popularity. Although I do appreciate the effort you put into them, it makes it easier to review.

**Comment by zrfisaac** [comment]
> Just letting you know @zrfisaac so you're not wasting time, we don't need a description of what the tool/brand does. We just need to know details related to assessing popularity. I do appreciate the effort you put into them though, the info makes reviewing easy.

Thanks!!!  I just search a good description from wikipedia or other source that match the real description, if it is good a put the description, so it not take so much time to do. 🙌

**Pull request merged**